packages/labs/context/src/test/context-provider_test.ts

```diff
@@ -136,3 +152,27 @@ suite('@consume: multiple instances', () => {
     );
   });
 });
+
+suite('@provide', () => {
+  test('can get values of provide decorated fields', async () => {
+    const container = document.createElement('div');
+    container.innerHTML = `
```

Comment:
we're already importing Lit, let's use it here, just to avoid issues later if we want to test with trusted types enabled